CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 * [CHANGE] Distributor: Remove `-distributor.enable-otlp-metadata-storage` flag, which was deprecated in version 2.12. #9069
 * [CHANGE] Ruler: Removed `-ruler.drain-notification-queue-on-shutdown` option, which is now enabled by default. #9115
 * [CHANGE] Querier: allow wrapping errors with context errors only when the former actually correspond to `context.Canceled` and `context.DeadlineExceeded`. #9175
+* [CHANGE] Query-scheduler: `-query-scheduler.prioritize-query-components` enables the querier-worker queue priority algorithm to take precedence over tenant rotation when dequeuing requests. #9220
```

Comment:
I would call this an `[ENHANCEMENT]` and note that the flag is experimental. Since it's default off, it's not a breaking change.

Reply:
done